CONTRIBUTORS.yaml

```diff
@@ -0,0 +1,252 @@
+# List of contributors
```

Comment:
Woah, that's really smart, I love it!

Reply:
README should reflect this change though, and maybe we have a link to this file in the website itself (not sure).

Reply:
ah yes, thanks, will update the README :)

And yes, was thinking of maybe making a little page for the website listing all the contributors, to give people some visibility/credit there as well as not everybody visits the github repo, and also because we have contributors who work more on infra than content and are not listed as contributors on any particular slides/tutorials, what do you think?

Reply:
Could we list everyone here maybe? Either in a new "Contributors" section, or something like "... to get started and join our Hall of Fame:" and the list of all the contributors. (could be something different than "Hall of Fame" too, but we need something appealing I'd say)

![screen shot 2017-09-06 at 14 45 01](https://user-images.githubusercontent.com/217628/30112559-10dfe848-9312-11e7-9d8f-ced1ecb87c5c.png)

Reply:
yes, good thought, I like this, will play around with this idea a bit sometime soon :)

Reply:
A slide show of github icons with the name under it ... :) 
Maybe 5 in a row and they are running from right to left ...

Reply:
ooh yes, nice idea :)